grid_problem.py

```diff
@@ -29,16 +39,122 @@ def get_cell_borders(i):
 
 class GridProblem:
     def __init__(self):
-        #populations = [int(random.random()*100 + 1) for i in range(w*h)]
+        # self.populations = [int(random.random()*100 + 1) for i in range(W * H)]
         self.populations = [1 for i in range(W*H)]
         self.areas = [1 for i in range(W*H)]
         self.borders = [get_cell_borders(i) for i in range(W*H)]
         self.num_districts = NUM_DISTRICTS
         self.total_population = reduce(lambda x, y: x + y, self.populations)
 
+    def split_district(self, district):
```

Comment:
I'm too lazy to read the rest lol, if it works it works and if it doesn't you'll fix it